networkit/cpp/distance/test/DistanceGTest.cpp

```diff
@@ -147,6 +152,91 @@ TEST_F(DistanceGTest, testAStar) {
     testMesh(25, 5);
 }
 
+TEST_P(DistanceGTest, testAdamicAdar) {
+    Aux::Random::setSeed(43, false);
+    auto G = generateERGraph(500, 0.03);
+    G.indexEdges();
+
+    AdamicAdarDistance AAD(G);
+    AAD.preprocess();
+    auto rndEdge = GraphTools::randomEdge(G);
```

Comment:
As in the comment in `testAlgebraicDistance`, just pick a fixed edge for the test.